scripts/template.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Webots Simulation Preview</title>
+    <link type="text/css" href="https://www.cyberbotics.com/jquery-ui/1.11.4/jquery-ui.min.css" rel="stylesheet" />
+    <link type="text/css" href="https://www.cyberbotics.com/wwi/R2020b/wwi.css" rel="stylesheet" />
+    <style>
+        html,
+        body {
+            padding: 0;
+            margin: 0;
+            height: 100%;
+            width: 100%;
+            display: flex;
+            flex-flow: row;
+        }
+
+        #view3d {
+            flex: 1;
+            background-color: black;
+        }
+
+        #sidebar {
+            padding: 12px;
+            max-width: 220px;
+        }
+
+        .simulation-item {
+            margin: 12px 0;
+        }
+
+        .simulation-item h3,
+        .simulation-item p {
+            margin: 0;
+        }
+    </style>
+    <script src="https://www.cyberbotics.com/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://www.cyberbotics.com/jquery-ui/1.11.4/jquery-ui.min.js"></script>
+    <script src="https://www.cyberbotics.com/jquery-dialogextend/2.0.4/jquery.dialogextend.min.js"></script>
+    <script src="https://www.cyberbotics.com/ace/1.2.0/ace.js"></script>
+    <script src="https://www.cyberbotics.com/wwi/R2020b/webots.min.js"></script>
+    <script>
+        const WORLD_LIST = { WORLD_LIST_PLACEHOLDER };
+        /*
+        const WORLD_LIST = [  
+            { description: 'Item description', title: 'Item title', name: 'world_name' },
+            { description: 'Item description', title: 'Item title', name: 'world_name' }
+        ];
+        */
+        var view = null;
+        var currentWorld = null;
```

Comment:
We usually prefer the `let` keyword to the `var` one because of its better scope properties.